tests/Schema/DataObject/Plugin/FiltersTest.php

```diff
@@ -109,4 +43,70 @@ private function testFilter(FieldFilterInterface $filter, string $identifier, ar
             );
         }
     }
+
+    public function filterProvider(): array
```

Comment:
```suggestion
    public function filterArgumentsProvider(): array
```
The convention tends to be that it matches the name of the test method that's using it like this.